### PR TITLE
MCLOUD-5535: Add service modification/validation tests

### PR DESCRIPTION
**Pull request by BaDos**
### Fixed Issues (if relevant)

https://jira.corp.magento.com/browse/MCLOUD-5535

### Manual testing scenarios

N/A

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: tests/functional/Codeception/BaseModule.php [hunk 1]
```diff
@@ -99,4 +105,46 @@ public function getArtifactsDir(): string
...
+     */
+    public function doNotSeeInOutput($text): void
+    {
+        if (is_array($text)) {
+            foreach ($text as $value) {
+                Assert::assertNotContains($value, static::$output);
+            }
+        } else {
+            Assert::assertNotContains($text, static::$output);
+        }
```
Comment (oshmyheliuk) [inline tests/functional/Codeception/BaseModule.php:138]: Can be shortened to 
```
            foreach (is_array($text) ? $text : [$text] as $value) {
                Assert::assertNotContains($value, static::$output);
            }
``` 
The same for the previous method.

**Review by oshmyheliuk — DISMISSED** [review]

File: src/Test/Integration/_files/cloud_base_mftf/composer.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "magento/project-enterprise-edition",
+  "description": "Composer file for integration tests",
+  "type": "project",
+  "version": "2.3.0"
+}
```
Comment (shiftedreality) [inline src/Test/Integration/_files/cloud_base_mftf/composer.json]: Missed line

File: src/Test/Integration/_files/cloud_base/composer.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "magento/project-enterprise-edition",
+  "description": "Composer file for integration tests",
+  "type": "project",
+  "version": "2.2.0"
+}
```
Comment (shiftedreality) [inline src/Test/Integration/_files/cloud_base/composer.json]: Missed line

**Review by shiftedreality — CHANGES_REQUESTED** [review]

File: tests/functional/Codeception/BaseModule.php (hunk 1 above)
Comment (BaDos) [reply tests/functional/Codeception/BaseModule.php:138]: I will change it in the next PR.
Thanks.

**Review by shiftedreality — DISMISSED** [review]

**Review by oshmyheliuk — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by oshmyheliuk** [comment]
Don't need QA as these changes are functional tests.

**Pull request merged**